pdc_client/__init__.py

```diff
@@ -109,6 +204,8 @@ class to perform requests.
         :param ssl_verify: True for validating SSL certificates with system CA
                            store; False for no validation; path to CA file or
                            directory to use for validation otherwise
+        :param is_multi:   This is about if the new object uses the MultiPageBeanBag
```

Comment:
Users of this API don't need to know about `MultiPageBeanBag` -- the class should be private and hidden from users.

More descriptive argument name would be nice, something like `get_all_pages` or `paged` (default `True`).

Reply:
I think it is ok to add the public class here, then user can choose the way to create codes they like and also this doesn't break the previous codes and logic.

Also I think is_multi is more directly  than the get_all_pages and paged, as this is a boolean

Reply:
> I think it is ok to add the public class here

Don't add stuff to public API unless really necessary. We need to make sure that changing the public API won't break existing code (otherwise users need to update them). And as I mentioned above `BeanBag.make_request()` is undocumented (not part of BeanBag's API?) so how do we know that we are re-implementing it correctly.

> Also I think is_multi is more directly than the get_all_pages and paged, as this is a boolean

Sorry but I cannot tell what `is_multi` means just by looking at the name.

Reply:
> Don't add stuff to public API unless really necessary. We need to make sure that changing the public API won't break existing code (otherwise users need to update them). And as I mentioned above BeanBag.make_request() is undocumented (not part of BeanBag's API?) so how do we know that we are re-implementing it correctly.

For this part I have made sure that this can't break the codes, and the make_request is the correct api of BeanBag, I will show you then in latter video/talk.

> Sorry but I cannot tell what is_multi means just by looking at the name.
If so I think, is_multi_page should be better, what's your thinking? and also this argument is described in docstring.